kmk/modules/layers.py

```diff
@@ -36,9 +36,13 @@ def __init__(self, layer, kc=None):
 class Layers(HoldTap):
     '''Gives access to the keys used to enable the layer system'''
 
-    def __init__(self):
+    def __init__(
+        self,
+        combolayers=None,
```

Comment:
general naming convention for variables is snake_case: `combo_layers`.

Reply:
fixed.